images/ceph/Makefile

```diff
@@ -17,11 +17,7 @@ include ../image.mk
 # ====================================================================================
 # Image Build Options
 
-ifeq ($(GOARCH),amd64)
```

Comment:
This is not uncommon to get the arm64 build later than the x86 one. So I think we should keep this. We have far less arm64 runners to produce the container builds.

Reply:
Okay understood

Reply:
@leseb I still have a doubt about keeping this if and else. The above point makes sense but that we need more amd64 runners but I guess this if and else check is just added for different ceph image versioning, and which is pointing to the same string now.

Reply:
https://github.com/rook/rook/blob/51d569eecc52421cf58791f4e2d55491eb2304f2/images/ceph/Makefile#L20